app/core/browse.py

```diff
@@ -76,7 +79,7 @@ def browse_rds_families(
     t0 = perf_counter_ns()
     geo_subquery = get_geo_subquery(db, req.geography_slugs, req.country_codes)
     query = (
-        db.query(Family, geo_subquery.c.value, Organisation)  # type: ignore
+        db.query(Family, Corpus, geo_subquery.c.value, Organisation)  # type: ignore
```

Comment:
Reckon it's worth building out this test seeing as the query is changed? 
https://github.com/climatepolicyradar/navigator-backend/blob/main/tests/core/test_browse.py#L8

Maybe an assertion of corpus fields being present.

Reply:
Good idea. Its quite a big change to that test but I'm happy to include it as a driveby here

Reply:
Done